lua/sh-tas-utils/copymodelproperty.lua

```diff
@@ -0,0 +1,18 @@
+
+AddCSLuaFile()
+
+properties.Add("copy_model", {
+	MenuLabel = "Copy Model",
+	Order = 2137,
+	MenuIcon = "icon16/page_copy.png",
+	Filter = function(self, ent, ply)
+		if not ent or not ent:IsValid() or ent:IsPlayer() then return end
+		return true
```

Comment:
nit: Can simplify this down to returning the condition
```suggestion
		return ent and ent:IsValid() and not ent:IsPlayer()
```